torch_geometric/nn/norm/agg_norm.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Dict, List, Optional, Union
+
+import torch
+from torch import Tensor
+
+from torch_geometric.nn.aggr import Aggregation
+from torch_geometric.nn.resolver import aggregation_resolver
+
+
+class AggSubtraction(torch.nn.Module):
+    r"""Applies layer normalization by subtracting an aggregation
+    over the inputs. Supports providing an aggregator from the
+    :class:`~torch_geometric.nn.aggr.Aggregation` module.
+
+    Args:
+        aggr (string or list or Aggregation, optional): The aggregation scheme
+            to use, *e.g.*, :obj:`"add"`, :obj:`"sum"` :obj:`"mean"`,
+            :obj:`"min"`, :obj:`"max"` or :obj:`"mul"`.
+            In addition, can be any
+            :class:`~torch_geometric.nn.aggr.Aggregation` module (or any string
+            that automatically resolves to it).
+            If given as a list, will make use of multiple aggregations in which
+            different outputs will get concatenated in the last dimension.
+            If set to :obj:`None`, the :class:`MessagePassing` instantiation is
+            expected to implement its own aggregation logic via
+            :meth:`aggregate`. (default: :obj:`"mean"`)
+        aggr_kwargs (Dict[str, Any], optional): Arguments passed to the
+            respective aggregation function in case it gets automatically
+            resolved. (default: :obj:`None`)
+    """
+    def __init__(
+        self,
+        aggr: Optional[Union[str, List[str], Aggregation]] = "mean",
+        aggr_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
I am not sure if any other option than mean makes any sense. It might also be that adding support for other aggregations yields to more confusion here as it suggests that this is a hyperparameter that can be actively tuned. I am in favor of just calling this module `MeanSubtractionNorm` which internally instantiates `MeanAggregator`. If you want to keep the flexible implementation, I will defer to you.

Reply:
I do agree with @rusty1s - and had the same concern once I'd implemented it. It might be more user-friendly to just have a `MeanAggregationNorm` and a `MedianAggregationNorm`.

@Alvaro-Ciudad WDTY? We could add it later if it ends up being useful, I agree with @rusty1s that user experience suffers with my approach.

Reply:
@Padarn I defer to your judgement. It is probably a better idea to remove it and modify it later if needed.

Reply:
Okay I'll update tomorrow morning. Thanks for the discussion.